Makefile

```diff
@@ -211,7 +211,7 @@ endef
 .PHONY: lint
 ## Checks the code with golangci-lint
 lint: $(GOLANGCI_LINT_BIN)
-	$(GOLANGCI_LINT_BIN) run -c .golangci.yaml --deadline=30m
```

Comment:
Question - is there another way to give this a time limit, now that the `deadline` flag is deprecated?

Reply:
Yes. It's already in [the golangci-lint config](https://github.com/openshift/external-dns-operator/blob/20c9abb024b5b7586ab25310d7ccddfc9aa3040f/.golangci.yaml#L9-L10). I don't see the reason to limit in 2 places. Also, `30m` was an overkill, this test should not take more than a couple of seconds.